app/Http/Controllers/UsersController.php

```diff
@@ -2,25 +2,102 @@
 
 namespace App\Http\Controllers;
 
+use App\Alexa\Models\User;
+use Illuminate\Http\Request;
+
 class UsersController extends Controller
 {
     /**
-     * Create a new controller instance.
+     * Display a listing of the resource.
+     *
+     * @param  \Illuminate\Http\Request  $request
      *
-     * @return void
+     * @return \Illuminate\Http\Response
      */
-    public function __construct()
+    public function index(Request $request)
     {
-        $this->middleware('auth');
+        $request->ajax() ?: abort(404);
```

Comment:
Why do you check if the request was the result of an AJAX call? I know that this is used when query members in the search bar to add members to a team, but can't we want to get a list of the users by some filter if we are *admins*? Don't know, just wondering :smile:

Reply:
And the goal is that. However, we do not have a use case for that, by now. 

My suggestion is to keep it like this and change on demand.

Reply:
:+1:

Reply:
👍